frontend/src/services/groupService.js

```diff
@@ -116,11 +116,13 @@ export const removeGroupCardService = (groupId) => {
 };
 
 export const setDisabledGroupService = (group) => {
-    group.disable = true;
+    const bufferGroup = group;
+    bufferGroup.disable = true;
     updateGroupService(group);
```

Comment:
technically, `group` and `bufferGroup` link to the same object, but `update...(bufferGroup)` seems more obvious